registration/forms.py

```diff
@@ -4,17 +4,18 @@
 import re
 import collections
 import datetime
+import requests
```

Comment:
is there a reason why you moved this? I tend to keep 3rd party libraries imported below django:

1. built ins imports
2. django imports
3. 3rd party library imports
4. project imports